lib/cartopy/crs.py

```diff
@@ -2038,12 +2032,38 @@ def y_limits(self):
 
 class Geostationary(_Satellite):
     """
+    A view appropriate for satellites in Geostationary Earth orbit.
+
     Perspective view looking directly down from above a point on the equator.
 
+    In this projection, the projected coordinates are scanning angles measured
+    from the satellite looking directly downward, multiplied by the height of
+    the satellite.
+
     """
     def __init__(self, central_longitude=0.0, satellite_height=35785831,
                  false_easting=0, false_northing=0, globe=None,
                  sweep_axis='y'):
+        """
+        Parameters
+        ----------
+            central_longitude: float, optional
```

Comment:
Unnecessary indent. It looks like there's no effect on the output, but I think that's because napoleon is a bit lenient and/or the theme settings.

Reply:
🐑 fixed